SingularityExecutorCleanup/src/main/java/com/hubspot/singularity/executor/cleanup/SingularityExecutorCleanup.java

```diff
@@ -215,7 +215,9 @@ private SingularityExecutorTaskDefinition withDefaults(SingularityExecutorTaskDe
             oldDefinition.getExecutorData().getRequestGroup(),
             oldDefinition.getExecutorData().getS3StorageClass(),
             oldDefinition.getExecutorData().getApplyS3StorageClassAfterBytes(),
-            oldDefinition.getExecutorData().getCpuHardLimit()
+            oldDefinition.getExecutorData().getCpuHardLimit(),
+            oldDefinition.getExecutorData().getHealthCheckResultFilePath().isPresent() ? oldDefinition.getExecutorData().getHealthCheckResultFilePath() : cleanupConfiguration.getHealthCheckResultFilePath(),
```

Comment:
cleanup shouldn't need to interact with the check at all, can just set it to either `oldDefinition.getExecutorData().get....` or just to absent

Reply:
This constructs a new `SingularityExecutorTaskDefinition` and I was following the pattern here. I would think just sending `Optional.absent()` would be okay too though. Thougths?

Reply:
absent would be fine here since it should be unused in this context